### PR TITLE
added the readme_generator_tool in the readme file

### LINKED ISSUE #583: ReadMe generator
**Is your feature request related to a problem? Please describe.**
Several developers or students build a portfolio website to showcase projects. But all the projects and stats are on Github. How about making your GitHub your portfolio?
Screenshot (161)

**Describe the solution you'd like**
I have a readme generator script that helps you to create ReadMe of your GitHub profile

**Describe alternatives you've considered**

**Additional context**
Pls look at the screenshot attached below

### PR DESCRIPTION
**Pull request by hr1shu**
# Description

Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change.

Fixes #583 

Replace `issue_no` with the issue number which is fixed in this PR

## Type of change

_Please delete options that are not relevant._

- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update
- [x] Documentation Update

# Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by hr1shu** [comment]
Please guide me to how to fix the  above issue

File: README.md [hunk 1]
```diff
@@ -59,6 +59,17 @@ You can refer to the following articles on the basics of Git and Github and also
...
+#### Tired of editing GitHub Profile README with new features?
+
+This tool provides an easy way to create a GitHub profile readme with the latest add-ons such as `visitors count`, `github stats`, etc.
+
+## 🚀 Demo 
+<a href="https://rahuldkjain.github.io/gh-profile-readme-generator" target="blank">
+<img src="https://img.shields.io/website?url=https%3A%2F%2Frahuldkjain.github.io%2Fgh-profile-readme-generator&logo=github&style=flat-square" />
+</a>
+
+Try the tool: [GitHub Profile README Generator](https://rahuldkjain.github.io/gh-profile-readme-generator)
```
Comment (HarshCasper) [inline README.md:71]: We don't need this. 

Maybe you had a very wrong conception around the Issue. I was perhaps looking for a Script that can generate a Profile README for the User and then push it to Github. We don't want our main README to be an advert for the tool. 

I'm sorry but I would not be able to merge the PR with this right here!

File: Python/Password Checker/Password-checker.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
+
```
Comment (HarshCasper) [inline Python/Password Checker/Password-checker.py:1]: Name of the Directory should be `Password_Checker` and not `Password Checker`

File: Python/Password Checker/Password-checker.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    count = pwned_api_check(password)
+    if count:
+      print(f'{password} was found {count} times... you should probably change your password!')
+    else:
+      print(f'{password} was NOT found. Carry on!')
+  return 'done!'
+
+if __name__ == '__main__':
+  sys.exit(main(sys.argv[1:]))
+
```
Comment (HarshCasper) [inline Python/Password Checker/Password-checker.py:37]: Remove the extra line of Code

File: Python/Password Checker/Password-checker.py [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  hashes = (line.split(':') for line in hashes.text.splitlines())
+  for h, count in hashes:
+    if h == hash_to_check:
+      return count
+  return 0
+
+def pwned_api_check(password):
+  sha1password = hashlib.sha1(password.encode('utf-8')).hexdigest().upper()
+  first5_char, tail = sha1password[:5], sha1password[5:]
+  response = request_api_data(first5_char)
```
Comment (HarshCasper) [inline Python/Password Checker/Password-checker.py:23]: Add proper Comments right here!

File: Python/Password Checker/Password-checker.py [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+  res = requests.get(url)
+  if res.status_code != 200:
+    raise RuntimeError(f'Error fetching: {res.status_code}, check the api and try again')
+  return res
+
+def get_password_leaks_count(hashes, hash_to_check):
+  hashes = (line.split(':') for line in hashes.text.splitlines())
+  for h, count in hashes:
+    if h == hash_to_check:
+      return count
```
Comment (HarshCasper) [inline Python/Password Checker/Password-checker.py:17]: Add proper Comments!

File: Python/Password Checker/Password-checker.py [hunk 6]
```diff
@@ -0,0 +1,37 @@
+
+import requests
+import hashlib
+import sys
+
+def request_api_data(query_char):
```
Comment (HarshCasper) [inline Python/Password Checker/Password-checker.py:6]: Add Docstrings to the functions.

File: Python/Password Checker/Password-checker.py [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+  url = 'https://api.pwnedpasswords.com/range/' + query_char
+  res = requests.get(url)
+  if res.status_code != 200:
+    raise RuntimeError(f'Error fetching: {res.status_code}, check the api and try again')
+  return res
+
+def get_password_leaks_count(hashes, hash_to_check):
```
Comment (HarshCasper) [inline Python/Password Checker/Password-checker.py:13]: Add Docstrings to the functions.

File: Python/Password Checker/Password-checker.py [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+  hashes = (line.split(':') for line in hashes.text.splitlines())
+  for h, count in hashes:
+    if h == hash_to_check:
+      return count
+  return 0
+
+def pwned_api_check(password):
```
Comment (HarshCasper) [inline Python/Password Checker/Password-checker.py:20]: Add Docstrings to the functions.

File: Python/Password Checker/README.md [hunk 9]
```diff
@@ -0,0 +1,20 @@
+# Password_Checker
+
+This self explanatory script, uses [PawnedPassword_API](https://pawnedpasswords.com).
+It checks how many times your password has been under hacaker radars.
+It's pretty cool.
```
Comment (HarshCasper) [inline Python/Password Checker/README.md:5]: Don't break your statement too often. Use full statements. And we definitely don't need "It's pretty cool" on the README. Follow the standard mentioned [here](https://github.com/HarshCasper/Rotten-Scripts/blob/master/TEMPLATE_README.md) and stick to the point.

File: Python/Password Checker/README.md [hunk 10]
```diff
@@ -0,0 +1,20 @@
...
+
+## Setup instructions and Guidelines
+
+This setup has only one requirement, `request`. Usually it is auto-downloaded with other packages.
+
+If that's not the case `pip  install request` will do the job.
```
Comment (HarshCasper) [inline Python/Password Checker/README.md:11]: Add a `requirements.txt` file and ask the User to install packages from there, for the purpose.

File: Python/Password Checker/README.md [hunk 11]
```diff
@@ -0,0 +1,20 @@
...
+
+
+The script has proper comments which should be followed. The script is easy to use.
```
Comment (HarshCasper) [inline Python/Password Checker/README.md:14]: Redundant! Please remove this.

File: Python/Password Checker/README.md [hunk 12]
```diff
@@ -0,0 +1,20 @@
...
+
+
```
Comment (HarshCasper) [inline Python/Password Checker/README.md:16]: Remove extra lines.

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Comment by hr1shu** [comment]
I'm sorry for the trouble mate.

**Comment by HarshCasper** [comment]
> I'm sorry for the trouble mate.

I don't understand the reason behind making and closing a PR after a review. You could have improved on it, at best. Anyways, Open-Source Contribution are all about voluntary contributions and learning out of the same ✌️

**Pull request closed without merging**